App Submission: Domain Locker

dennysubke opened the pull request:
# App Submission

### Domain Locker
...

### Icon
![logo](https://github.com/user-attachments/assets/7123d411-be5f-44d9-a6d0-070c7b5cdb6f)

...

### Gallery images
![1](https://github.com/user-attachments/assets/f84951f1-b42b-44f2-be50-c90dc5078792)

![2](https://github.com/user-attachments/assets/d57cebc4-8579-493a-bea7-0ac50d3f061d)

![3](https://github.com/user-attachments/assets/961a80ec-94c4-4bfb-b4ee-573ac80f34e6)

![4](https://github.com/user-attachments/assets/57e265dd-a678-4a87-8906-07e44e709a36)

![5](https://github.com/user-attachments/assets/db524dd4-e7d4-49b8-a8f5-5f5588794bd9)

### I have tested my app on:
- [ ] umbrelOS on a Raspberry Pi
- [X] umbrelOS on an Umbrel Home
- [X] umbrelOS on Linux VM

al-lac commented on domain-locker/docker-compose.yml:
```diff
@@ -0,0 +1,32 @@
+      DL_PG_HOST: ${DL_PG_HOST:-postgres}
+      DL_PG_PORT: ${DL_PG_PORT:-5432}
+      DL_PG_USER: ${DL_PG_USER:-postgres}
+      DL_PG_PASSWORD: ${DL_PG_PASSWORD:-changeme2420}
```
I think you can remove the variable names here as those will not be used anyways. And maybe change the value to something else besides `changeme`.
```suggestion
      DL_PG_PASSWORD: domain-locker

```

al-lac commented on domain-locker/docker-compose.yml:
```diff
@@ -0,0 +1,32 @@
+      - postgres
+
+  postgres:
+    image: postgres:15-alpine@sha256:ef9d1517df69c4d27dbb9ddcec14f431a2442628603f4e9daa429b92ae6c3cd1
```
Maybe we could try a more up-to-date version of postgres here like 17.4, just so we don't have to update this soonish.

al-lac commented on domain-locker/docker-compose.yml:
```diff
@@ -0,0 +1,32 @@
+    user: "1000:1000"
+    restart: on-failure
+    environment:
+      POSTGRES_DB: ${DL_PG_NAME:-domain_locker}
+      POSTGRES_USER: ${DL_PG_USER:-postgres}
+      POSTGRES_PASSWORD: ${DL_PG_PASSWORD:-changeme2420}
```
Would also remove the variable names here and adapt the password.

al-lac reviewed (changes requested):
Hey @dennysubke!

Another great app submission, you are on a roll! 🚀 

Only have some minor points, tested the app and it works. Sometimes I had to reload pages for them to show something, this could also be due to the app still being in pretty fresh and being more like a beta version.

dennysubke commented:
Moin @al-lac! 

Thanks again for the feedback! I’ve addressed all your suggestions. Everything should be good to go now. 😊 

Let me know if there’s anything else!
